### DISCUSSION TITLE
Direct Affil CLAIM button keeps Green after Claiming and wont reset to 0

### DISCUSSION THREAD

**Discussion by bf-mike-mkt** [betfinio/public#73] [Bugs and Reports]
<img width="524" alt="image" src="https://github.com/user-attachments/assets/b433088b-67db-4bbd-ac8c-8fad9a3f2836">

**Comment by bf-mike-mkt** [comment]
saw it on more than 5 accounts

**Comment by bf-mike-mkt** [comment]
Additionaly, Notification says: Claiming MATCHING bonus. (I think it should say claiming Direct bonus)

**Comment by bf-monster** [comment]
noted, thank you

**Comment by bf-mike-mkt** [comment]
Still happening + notification says "matching bonus" its should say "Direct bonus"
